content/client-server-api/_index.md

```diff
@@ -1831,6 +1841,47 @@ the topic to be removed from the room.
 
 {{% http-api spec="client-server" api="redaction" %}}
 
+### Forming relationships between events
+
+{{% changed-in v="1.3" %}}
+
+In some cases it is desirable to logically associate one event's contents with
+another event's contents. For example, when replying to a message, editing an
+event, or simply looking to add context for an event's purpose.
+
+Relationships are defined as part of an event's `content`. Any event can relate
+to any other event, however the relationship itself might have restrictions
+depending on its `rel_type`. Those restrictions are described by the relationship
+type in this specification, if any exist.
+
+The relationship is stored under the `m.relates_to` key of `content`, referencing
+the "parent" event. Both the event with `m.relates_to` and the event targeted by
+`m.relates_to` MUST exist in the same room.
+
+{{% boxes/note %}}
+For simplicity, a single type of relationship is permitted on an event at a time.
+A future MSC might change this if a use case arises.
+{{% /boxes/note %}}
+
+{{% boxes/note %}}
+An event with `m.relates_to` can relate to another event with `m.relates_to`,
+forming a sort of chain of events.
+{{% /boxes/note %}}
+
+`m.relates_to` is described as follows:
+
+{{% definition path="api/client-server/definitions/m.relates_to" %}}
+
+### Relationship types
+
+This specification describes the following relationship types:
+
+* [Rich replies](#rich-replies)
```

Comment:
It seems a little weird to include this here as rich replies use a different format:

```json
{
  "m.relates_to": {
    "m.in_reply_to": {
      "event_id": "$another_event"
    }
}
```

vs.

```json
{
  "m.relates_to": {
    "rel_type": "whatever",
    "event_id": "$another_event"
  }
}
```

Reply:
noted, however this is carryover from older versions of the spec. Will play with wording to try and clarify what's going on.